src/main/webapp/app/stores/firebase/firebase-app.store.ts

```diff
@@ -52,15 +52,19 @@ export class FirebaseAppStore extends BaseStore {
   }
 
   initializeFirebase() {
-    const { enabled, ...firebaseOptions } = AppConfig.serverConfig.frontend.firebase;
+    const { enabled, isTest, ...firebaseOptions } = AppConfig.serverConfig.frontend.firebase;
     this.firebaseEnabled = enabled;
     if (this.firebaseEnabled) {
       try {
         this.firebaseOptions = firebaseOptions as FirebaseOptions;
         this.firebaseApp = initializeApp(this.firebaseOptions);
         this.firebaseDb = getDatabase(this.firebaseApp);
-        const auth = getAuth();
         this.firebaseInitSuccess = true;
+
+        const auth = getAuth();
+        if (isTest) {
+          connectAuthEmulator(auth, 'http://127.0.0.1:9099', { disableWarnings: true });
```

Comment:
Is it possible to mock this logic into a class or function so you don't have to include the `isTest` flag?